rever/activities/push_tag.xsh

```diff
@@ -31,6 +32,11 @@ class PushTag(Activity):
                          desc="Tags the current version.")
 
     def _func(self, remote=None, target=None):
+        if 'TAG_REMOTE' in ${...}:
+            warnings.warn(DeprecationWarning('TAG_REMOTE has been deprecated '
```

Comment:
I don't think that this follows the API of warn(). https://docs.python.org/3.6/library/warnings.html#warnings.warn